src/components/TagEditor/TagEditor.tsx

```diff
@@ -28,7 +33,15 @@ export function TagEditor({tags, tagSuggestions, onInput, onAdd, onDelete, mes,
 
   function onValidate(tag: Tag) {
     let valid = true;
-    tags.forEach(t => {if (t.name === tag.name) valid = false;});
+    if (filter.isProfane(tag.name) ||
+        tag.name.charAt(0) === tag.name.charAt(0).toLowerCase() ||
```

Comment:
Instead of restricting a tag from being added if it starts with a lowercase letter, just automatically capitalize the the first letter.